test/perf/regression/resources/etcd/pod.yaml

```diff
@@ -10,10 +10,14 @@ spec:
   - name: etcd
     image: {{ .etcdImage }}
     command:
-    - "/bootstrap/bootstrap.sh"
+    - "/var/etcd/bin/bootstrap.sh"
     env:
     - name: GODEBUG
       value: gctrace=1
+    - name: ENABLE_TLS
+      value: "false"
+    - name: BACKUP_ENDPOINT
+      value: "http://localhost:8080"
```

Comment:
For my information, what are these for?

Reply:
This pod yaml is for the perf test, and since I've changed the etcd version to `v3.4.13-bootstrap`, I've also removed bootstrap script from the configmap and passed env vars to the bootstrap script residing on the etcd custom image.